app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Brand.kt

```diff
@@ -7,6 +7,7 @@ import androidx.room.PrimaryKey
 /**
  * @param bid: DB ID for dev use
  * @param brandName: User Inputted Name
+ * @param brandUrl: Brand URL
```

Comment:
Missing parameter comments for 'brandUrl'.

Reply:
When you add the @param tag, describe the parameter as well.  One could read the parameter name from the function signature; KDoc can add value to this.